config.json

```diff
@@ -18,6 +18,9 @@
   "test_runner": {
     "average_run_time": 3.0
   },
+  "approaches": {
+    "snippet_extension": "hs"
+  },
```

Comment:
We don't yet support this in the website, sorry. The approach snippets must thus all be `snippet.txt`.

Side note: I wonder if making approach snippets be `snippet.md` markdown files like with articles is the more sensible option.